scripts/test_backref_cost_specialization_factorization_v1_experiment.py

```
@@ -0,0 +1,248 @@
+RUNNER = "backref_cost_specialization_factorization_v1_experiment_runner"
+
+
+def run(argv: list[str], *, env: dict[str, str] | None = None,
+        check: bool = True) -> subprocess.CompletedProcess[str]:
+    clean = os.environ.copy()
+    for name in ("WEBP_BENCHMARK_SESSION", "CUDA_VISIBLE_DEVICES",
+                 "MallocNanoZone", RUNTIME):
+        clean.pop(name, None)
+    if env:
+        clean.update(env)
+    result = subprocess.run(argv, cwd=ROOT, env=clean, text=True,
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)